st_cc.py

```diff
@@ -164,6 +164,35 @@ def get_opt(view):
     project_settings = view.settings()
     include_opts = settings.get("include_options", []) + project_settings.get("cc_include_options", [])
 
+    # seach for .clang_complete file and add lines to options
+    target_file = ".clang_complete"
+    directory = os.path.dirname(view.file_name())
```

Comment:
Thanks for checking it over @trishume

I agree about taking into account the window's folder - it might be best stop the search for .clang_complete when it reaches the root of the folder that's associated with both the window and the file, using sublime.active_window().folders() would work but I think we'd need to search all of them so we choose the folder that the file is buried within

Searching up the entire directory to / work but there's some edge cases where it could pick up a .clang-complete you didn't intent it to use